content/en/news/releases/1.18.x/announcing-1.18/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Announcing Istio 1.18.0
+linktitle: 1.18.0
+subtitle: Major Release
+description: Istio 1.18 Release Announcement.
+publishdate: 2023-06-02
+release: 1.18.00
+aliases:
+    - /news/announcing-1.18.0
+---
+
+We are pleased to announce the release of Istio 1.18. This is the second Istio release of 2023, and the first to ship with Ambient mode! We would like to thank the entire Istio community for helping get the 1.18.0 release published. We would like to thank the Release Managers for this release, `Paul Merrison` from Tetrate, `Kalya Subramanian` from Microsoft and `Xiopeng Han` from . The release managers would specially like to thank the Test & Release WG lead Eric Van Norman (IBM) for his help and guidance throughout the release cycle. We would also like to thank the maintainers of the Istio work groups and the broader Istio community for helping us throughout the release process with timely feedback, reviews, community testing and for all your support to help ensure a timely release.
```

Comment:
Do we want to call out stability level of Ambient?

Reply:
I know we have plenty of users asking about eBPF and Ambient. Should we say anything about the licensing issue we are dealing with?

Reply:
@howardjohn Can you chime in here on Eric and Jacob's questions?